.coveragerc

```
@@ -2,6 +2,7 @@
 omit = 
     .nox/*
     *tests*
+    plugins/hydra_ax_sweeper/*
```

If you disabled it for plugins, go all the way.

```suggestion
    plugins/*
```

---

Done